src/methods/clipping/difference.jl

```diff
@@ -0,0 +1,292 @@
+# #  Difference Polygon Clipping
+export difference
+
+# The 'difference' function returns the difference of two polygons. Note that this file
+# currently only contains the difference of two polygons, which will always return
+# a vector of a vector of a vector of tuples of floats.
+
+"""
+    _trace_difference(::GI.PolygonTrait, poly_a,
+     ::GI.PolygonTrait, poly_b, a_list, b_list, a_idx_list,
+      intr_list, edges_a, edges_b)::Vector{Vector{Tuple{Float64}}}, Bool
+
+Traces the outlines of two polygons in order to find their difference.
+It returns the outlines of all the components of the difference. The Bool
+indicates whether or not these components (each Vector{Tuple{Float64} inside
+the larger Vector) are part of the same polygon (true) or each different
+polygons (true).
+
+## Example
+
+FILL IT IN
```

Comment:
You don't need an example for this. It is okay. That is really for the user-facing functions